charms/katib-controller/terraform/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  default     = "katib-controller"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = null
```

Comment:
Should we default the channel to 0.17/stable?

Reply:
updated